modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -0,0 +1,90 @@
+
+You can add API instances to the API Group instance based on the environment (Production, Sandbox, or others) in which you create the API Group instance. For example, if you created a API Group instance in Production environment, you can add API instances to the API Group instance only from Production.
+
+You create API Groups using the following steps:
+
+*Step 1:* <<create-api-groups,Create an API Group>>
+
+The first step to creating an API Group that can be used by consumers, is create an API Group. 
+
+*Step 2:* <<adding-api-instances-to-groups,Add API instances to the new API Group>>
+
+After you create the API Group, you then add API instances to this group for the first time.
+
+== Task Prerequisites
+
+Before you begin, you must have:
+
+* At least one API instance in your organization or business group.
+// +
+// Otherwise, you may not find the required organization or business group in the list.
+* API Group Administrator permissions to perform to create API Groups .
+//Group administrators is a permission which is set a business group *BG) level. A group administrator can manage (create, update, delete) all groups of the BG. The permissions algo, gives user VIEW access to all API instances of the BG (so user can add them to the group)
+
+
+[[create-api-groups]]
+== Create API Groups
+
+You create an API Group at the organization level.
+
+To create API Groups:
+
+. From the left menu in API Manager, click *API Groups > Create API Groups*.
++
+The Create New API Group window appears.
+. In the *API Group Name* field, specify a name for the API Group that you want to create: for example, `Social Media`.
+. In the *API Group version* field, specify the version for the API Group, for example, `v1`.
+. Optionally, in the API Group instance label field, specify a label for the API Group, such as `Internal`, if it is to be used for internal purposes.
+. Click *Ok*.
++
+The *Creating a new group* page is displayed. You are now ready to <<adding-api-instances-to-groups,add API instances>> to the API Group.
+
+
+[[adding-api-instances-to-groups]]
+== Add API instances to API Groups
+
+Your API Group is not complete and functional unless you add the API instances to the group that you want to bundle. When you add API instances to an API Group, you can add:
+
+* Only those API instances that belong in the same business group or organizations.
+* Only those API instances that belong the same environment as the API Group instance.
+* Only those API instances that belong to the same identity provider (IdP) as the API Group instance. You cannot mix IdPs in an API Group instance.
+
+To add a new API instance to a new API Group:
+
+. On the *Creating a new group* page, select an environment from the *Select environment* drop-down list.
++
+.The screen capture shows how to add API instance to an API Group.
+image:create-api-instances-api-groups.png[Adding API instances to API Groups]
++
```

Comment:
NO PERIODS and format UI element labels.

Reply:
Done! Thanks!